### PR TITLE
Update LessCompiler.cs

### PR DESCRIPTION
**Pull request by drphrozen**
Removed duplicate argument

### CODE REVIEW

File: src/WebCompiler/Compile/LessCompiler.cs [hunk 1]
```diff
@@ -121,9 +121,6 @@ private static string ConstructArguments(Config config)
             if (!options.IECompat)
                 arguments += " --no-ie-compat";
 
-            if (options.StrictMath)
```
Comment (madskristensen) [inline src/WebCompiler/Compile/LessCompiler.cs]: I don't see this being duplicated anywhere?

Comment (drphrozen) [inline src/WebCompiler/Compile/LessCompiler.cs]: The first entry is at line 118 (hidden in the diff), either expand above or checkout here:
https://github.com/madskristensen/WebCompiler/blob/master/src/WebCompiler/Compile/LessCompiler.cs#L118

**Comment by markvantilburg** [comment]
it's there twice indeed

File: src/WebCompiler/Compile/LessCompiler.cs (hunk 1 above)
Comment (304NotModified) [inline src/WebCompiler/Compile/LessCompiler.cs]: 118 + 124 are indeed duplicates:

https://github.com/madskristensen/WebCompiler/blob/master/src/WebCompiler/Compile/LessCompiler.cs#L118-L125

**Comment by 304NotModified** [comment]
why isn't this merged?

**Pull request merged**